Makefile

```diff
@@ -151,16 +151,21 @@ clean-glide-cache:
 deps: prebuild-check
 	$(GLIDE_BIN) --verbose install
 
-.PHONY: generate
-## Generate GOA sources. Only necessary after clean of if changed `design` folder.
-generate: prebuild-check $(DESIGNS) $(GOAGEN_BIN) $(GO_BINDATA_ASSETFS_BIN) $(GO_BINDATA_BIN)
+app/controllers.go: $(DESIGNS) $(GOAGEN_BIN)
```

Comment:
app/controller.go is one of the files that are generated. It doesn't matter if the content is changed or not, as it's deleted and recreated if 'anything' change. So it will always have a new timestamp if design/*.go change.